recipes/maesters-nwp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "maesters-nwp" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 91165b891d4d1707e08572af996080fe2376a2978f4c963203b7246bc4e8d245
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+
+  run:
+    - setuptools
```

Comment:
```suggestion
```
setuptools is not listed in the [run requirements upstream](https://github.com/blizhan/Maesters-of-NWP/blob/main/requirements.txt).